config/default.go

```diff
@@ -148,6 +148,10 @@ scrollbar = "auto"
 # bind_reload
 # bind_back
 # bind_forward
+# bind_moveup
+# bind_movedn
+# bind_moveleft
+# bind_moveright
```

Comment:
Remove these from this file. Add them to `default-config.toml` instead, and then run `go generate ./...`. Sorry this is not documented.

Reply:
I have added that to the `default-config.toml` `go generate ./...` generates this default.go so I guess no changes are required here.

Reply:
Yep, seems good then.